Add support for multi-AZ configurations of DBaaS clusters

paaanic opened the pull request:
VKPRGM-2010

ftersin commented on CHANGELOG.md:
```diff
@@ -9,6 +9,7 @@ description: |-
 
 #### v0.9.3 (unreleased)
 - Add secure_key option to cdn resource
+- Add availability_zones and vrrp_port_id fields to vkcs_db_cluster resource
```
could we also mention multi-az postgre directly?

ftersin commented on vkcs/db/resource_cluster.go:
```diff
@@ -179,6 +179,12 @@ func ResourceDatabaseCluster() *schema.Resource {
+			"vrrp_port_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The id of the VRRP port attached to the cluster. __note__  Only available in multi-AZ configurations.",
```
extra space after note keyword

paaanic replied:
Removed.

paaanic commented:
> an example is required for new datastore
> probably you could convert acceptance test to the example

@ftersin Removed the mentioned acceptance test and replaced it with the corresponding example, added it to docs as well.
